Description: Display session-properties in the menu
 gnome-control-center no longer includes the Startup Applications capplet,
 so at least make session-properties discoverable via the menu or via
 the builtian search.
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=683814

Index: gnome-session-3.4.2.1/data/session-properties.desktop.in.in
===================================================================
--- gnome-session-3.4.2.1.orig/data/session-properties.desktop.in.in	2012-05-17 20:26:07.000000000 +0200
+++ gnome-session-3.4.2.1/data/session-properties.desktop.in.in	2012-08-07 01:08:23.755210722 +0200
@@ -8,7 +8,6 @@
 StartupNotify=true
 Categories=GTK;GNOME;Settings;X-GNOME-PersonalSettings;
 OnlyShowIn=GNOME;Unity;
-NoDisplay=true
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-session
 X-GNOME-Bugzilla-Component=gnome-session-properties
